layouts/partials/home/cncf.html

```diff
@@ -2,7 +2,7 @@
 <section class="section">
   <div class="container has-text-centered">
     <p class="title is-size-3 is-size-4-mobile has-text-weight-light">
-      etcd is a <a href="https://cncf.io">Cloud Native Computing Foundation</a> incubating project
+      etcd is a <a href="https://cncf.io">Cloud Native Computing Foundation</a> graduated project
```

Comment:
```suggest
etcd is a <a href="https://www.cncf.io">Cloud Native Computing Foundation</a> project.
```
Since the project has graduated, it doesn't need to specify that it's level anymore :)
https://github.com/cncf/foundation/blob/master/website-guidelines.md